schema.graphql

```diff
@@ -65,6 +67,22 @@ type NftContractOwner @entity {
   quantity: BigInt!
 }
 
+type NftContractDayData @entity {
+  # {contractAddress}-{dateInEpochTime}
+  id: ID!
+  contract: NftContract!
+  epochTime: Int!
```

Comment:
is there a better name for this? open to suggestions..

Reply:
how about just good old `dateTime`